src/modules/Dashboard/features/Applet/Schedule/EventForm/NotificationsTab/Notification/Notification.tsx

```diff
@@ -40,22 +46,34 @@ export const Notification = ({ index, remove }: NotificationProps) => {
           <StyledLeftCol>
             <ToggleButtonGroup
               toggleButtons={notificationTimeToggles}
-              activeButton={notification.type}
+              activeButton={notification.triggerType}
               customChange={updateTime}
             />
           </StyledLeftCol>
           <StyledCol sx={{ marginLeft: theme.spacing(2.4) }}>
-            {notification.type === NotificationType.Fixed ? (
+            {notification.triggerType === NotificationType.Fixed ? (
               <StyledColInner>
-                <TimePicker name={`notifications.${index}.at`} label={t('at')} />
+                <TimePicker
+                  providedValue={atTime}
```

Comment:
Why used providedValue ?

Reply:
added TODO to fix in the future in the TimePicker.tsx